### PR TITLE
Update to 1.16, configure Nix and Lorri

### LINKED ISSUE #5: Any plans to update?
I'm interested to use this with k8s >=1.15

### PR DESCRIPTION
**Pull request by yurrriq**
Looks like I was able to update to 1.16, and I took the liberty of configuring a Nix shell and generating some edocs... Let me know what you think.

Close #5.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+gen: swagger.json
+	@ kuberl_gen
```
Comment (tsloughter) [inline Makefile:3]: Did you forget to add a file to the commit? I don't see a kuberl_gen anywhere.

**Comment by tsloughter** [comment]
So lorri is part of the nix stuff?

File: Makefile (hunk 1 above)
Comment (yurrriq) [reply Makefile:3]: `kuberl_gen` comes from here: https://github.com/tsloughter/kuberl/blob/1d4f48b0f85c2938f78542f36ee0b1b18889cee3/shell.nix#L4-L17

**Comment by yurrriq** [comment]
Yeah, lorri + direnv compares to something like virtualenv. Basically it builds the `shell.nix` expression and adds a garbage collection root to the Nix store. The main downside to this MR is that it requires full Nix buy-in :confused:.

**Comment by tsloughter** [comment]
Oh, yea... I'm fine with adding the nix stuff but not it being required. Are you able to make the generation stuff work without nix and also not be so separate that if one script changes they don't end up out of sync?

**Comment by yurrriq** [comment]
Yeah, I can do that. I'll update the README to reflect Nix-based and Nix-less workflows, and make sure the script works either way.

**Comment by yurrriq** [comment]
Ok, should be cool now.

**Comment by tsloughter** [comment]
Nice, thanks!

**Pull request merged**